java/buildconf/ivy/obs-maven-config.yaml

```diff
@@ -8,6 +8,9 @@ repositories:
   Uyuni:
     project: systemsmanagement:Uyuni:Master
     repository: openSUSE_Leap_15.2
+  Home:
+    project: home:j_renner:branches:systemsmanagement:Uyuni:Master:Other
+    repository: openSUSE_Leap_15.2
```

Comment:
Don't we have that jade in a more official repo before merging that PR?

Reply:
Of course, the idea was to take it from the home project only to see if CI is passing, see also the PR description. I was planning to SR the package to the official project next week in case everything looks good (e.g. unit tests).